api-definitions/db_positions/api.yaml

```diff
@@ -326,6 +343,29 @@ definitions:
           - F
           - G
         description: Type of person (F: PERSON_NATURAL, G: PERSON_LEGAL)
+      street_name:
+        type: string
+        example: Via Gonin
+      civic_number:
+        type: string
+        description:
+          type: string
+          example: 36
+      postal_code:
+        type: string
+        example: 20124
+      city:
+        type: string
+        example: Milano
+      state_province_region:
+        type: string
+        example: Mi
+      country:
+        type: string
+        example: Italia
+      email:
+        type: string
+        example: example@sia.eu
```

Comment:
For all fields where is possibile to use regex or format hint, we shall to use it.  see [here](https://swagger.io/docs/specification/data-models/data-types/)

Reply:
Ok, we added format:email in the field. We found no other regex to be added, but if you find any, please let us know.